ci/kokoro/macos/builds/cmake-vcpkg.sh

```diff
@@ -101,10 +109,9 @@ if should_run_integration_tests; then
     export CLOUD_STORAGE_ENABLE_TRACING="raw-client"
 
     cd "${BINARY_DIR}"
```

Comment:
Ditto on `env -C`.  We could even use `env` to do all those `export`s.

Reply:
I moved the exports, but we cannot do the -C